src/app/Components/GoogleCallback.tsx

```
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { toast } from 'sonner';
+import { useGoogleLogin } from '../hooks/useAuth';
+import { Mutation } from '@tanstack/react-query';
```

ESLint

[error] 'Mutation' is defined but never used.

(@typescript-eslint/no-unused-vars)